docs/source/reference/ui.rst

```diff
@@ -0,0 +1,78 @@
+.. _ui:
+
+User interface manual
+=====================
+
+Page transition flow::
+
+  [Home: Project list]
+    -- (select project) ->
+      [Project: Show training chart and jobs]
+        -- (select result ID) ->
+          [Result: Show detail information of result]
+
+Header
+------
+
+.. image:: ../../images/header_green.png
+
+.. |header_config| image::  ../../images/header_config.png
+   :scale: 50%
+
+.. |header_status_green| image:: ../../images/header_status_green.png
+   :scale: 50%
+
+* |header_config| : setup global configuration and show ChainerUI version:
+    * select size of a chart
+    * select polling interval
+    * show ChainerUI version
+* |header_status_green| : connection status between ChainerUI server
+    * green: success to connect
+    * blue: loading
+    * red: fail to connect
+    * gray: disable polling
+
+Home: Project list
+------------------
+
+.. image:: ../../images/home_project_list.png
+
+List of registered projects, select a project, transition to the project page. When register a project within running server, refresh the page and will show the project on the list. See :ref:`getstart_customize_training_loop`.
```

Comment:
From the list of registered projects, select a project to transition to the project page. When registering a project... and it will show...